admin_manual/configuration/server/security/ransomeware-protection.rst

```diff
@@ -0,0 +1,95 @@
+There are known attacks that change file extensions randomly or keep them unchanged which makes them harder to detect.
+
+Ultimately there is a consensus that only one solution can provide future-proof protection from ransomware attacks: retaining data and providing the means to roll back to a particular point in time.
+
+ownCloud Ransomware Protection will, therefore, record all changes on an ownCloud Server and allow administrators to rollback user data to a particular point in time, making use of ownCloud’s integrated Versioning and Trash bin features.
+
+Doing so allows all user data that is synchronized with the server to be rolled back to its state before the attack occurred.
+We expect that a combination of Ransomware prevention and protection should reduce risks to a minimum acceptable level.
+
+Other Elements of Ransomware Protection
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+==================== ============================================== ================================================
+Name                 Command                                        Description
+==================== ============================================== ================================================
+Ransomguard Scanner  ``occ ransomguard:scan <timestamp> <user>``    A command to scan the ownCloud database for
+                                                                    anomalies in a user account. It enables an administrator to determine the point in time
+                                                                    where undesired actions happened as a
+                                                                    prerequisite for restoration.
+Ransomguard Restorer ``occ ransomguard:restore <timestamp> <user>`` A command for administrators to revert all
+                                                                    operations in a user account that occurred after
+                                                                    a certain point in time.
+==================== ============================================== ================================================
+
+.. note:: 
+   ``<timestamp>`` must be in `the Linux timestamp format`.
+
+Requirements
+~~~~~~~~~~~~
+
+#. **Ransomware Protection.** It records the move operations so that it can revert them in the case of an attack. Other operations are revertible using ownCloud's existing database tables.
+#. **ownCloud Versions App.** It is required to restore older file versions and because capabilities of Ransomware Protection depend on its configuration.
+#. **ownCloud Trash Bin App.** It is required both to restore deleted files and because capabilities of Ransomware Protection depend on its configuration.
+
+Limitations
+~~~~~~~~~~~
+
+-  Ransomware Protection works with master-key based storage encryption. Credential-based storage encryption only Ransomware Prevention (Blocking) works.
```

Comment:
"Credential-based storage encryption only..." ?
maybe it means:
"With credential-based storage encryption only..." ?
and is "Credential-based storage encryption" the inidividual user-by-user encryption?

Reply:
What is meant here is the storage encryption type that is based on user passwords in contrast to master-key based storage encryption. So at least a "with" is missing here.

Reply:
Commented in conversation.

Reply:
Thanks. I wasn't sure about that sentence.